### PR TITLE
FEAT: Account Managment Screen

### PR DESCRIPTION
**Pull request by duolok**
This PR adds a screen for Account Management. Users can change their profile info, update their password, and delete their account from here.
demo_three
demo_two
demo_one

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -15,4 +15,11 @@
     <string name="welcome_to_lunark">Welcome to Lunark!</string>
     <string name="i_am_a_guest">I am a guest</string>
     <string name="i_am_a_host">I am a host</string>
+
+    <string name="title_activity_account_screen">AccountScreen</string>
+    <string name="header_your_profile">Your Profile</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (vradovic) [inline app/src/main/res/values/strings.xml]: Remove this, it is already defined on line 9

**Review by vradovic — CHANGES_REQUESTED** [review]

**Review by vradovic — CHANGES_REQUESTED** [review]
The app launches from SplashScreen activity to the Account activity, it should probably launch to Home or Login activity instead :thinking: 

Also selecting Account from the navigation drawer doesn't open the Account screen

**Comment by duolok** [comment]
> The app launches from SplashScreen activity to the Account activity, it should probably launch to Home or Login activity instead

Testing purposes.

> Also selecting Account from the navigation drawer doesn't open the Account screen

It will be updated in another PR.

**Comment by vradovic** [comment]
Resolve conflicts as well please :pray:

**Review by vradovic — APPROVED** [review]

**Pull request merged**